File: Beecrowd/1555 Funções/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int n, x, y, r, b, c, maior;

        n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            x = sc.nextInt();
            y = sc.nextInt();

            r = (int) (Math.pow(3 * x, 2) + Math.pow(y, 2));
            b = (int) ((2 * Math.pow(x, 2)) + Math.pow(5 * y, 2));
            c = (int) ((-100 * x) + Math.pow(y, 3));

            if (r > b && r > c)
                maior = r;
            else if (b > r && b > c)
                maior = b;
            else
                maior = c;

            if (maior == r)
                System.out.println("Rafael ganhou");
            else if (maior == b)
                System.out.println("Beto ganhou");
            else
                System.out.println("Carlos ganhou");
        }

        sc.close();
    }

}

File: Beecrowd/1555 Funções/main.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main()
{
    int N, i, x, y, r, b, c, maior;

    cin >> N;

    for (i = 0; i < N; i++)
    {
        cin >> x >> y;
        r = pow(3 * x, 2) + pow(y, 2);
        b = (2 * pow(x, 2)) + pow(5 * y, 2);
        c = (-100 * x) + pow(y, 3);
        if (r > b && r > c)
            maior = r;
        else if (b > r && b > c)
            maior = b;
        else
            maior = c;

        if (maior == r)
            cout << "Rafael ganhou" << endl;
        else if (maior == b)
            cout << "Beto ganhou" << endl;
        else
            cout << "Carlos ganhou" << endl;
    }

    return 0;
}